setup.cfg

```diff
@@ -33,6 +33,7 @@ install_requires =
     # this is a temporary dependency, we published the repo https://github.com/o19s/solr-to-es to pypi ourselves
     # until the ticket https://github.com/o19s/solr-to-es/issues/23 is resolved.
     pds.solr-to-es==0.3.0
+    tqdm==4.66.2
```

Comment:
Consider using `~=` to get bug-fixes and security-udpates to `tqdm` "for free"?

Reply:
done